src/wallet/rpc/spend.cpp

```diff
@@ -78,6 +58,36 @@ static void InterpretFeeEstimationInstructions(const UniValue& conf_target, cons
     }
 }
 
+std::set<int> InterpretSubtractFeeFromOutputInstructions(const UniValue& options, const std::vector<std::string>& destinations)
+{
+    int pos{0};
+    std::set<int> sffo_set;
+    UniValue subtract_fee_outputs(UniValue::VARR);
+    if (options.exists("subtractfeefromamount") || options.exists("subtractfeefrom")) {
```

Comment:
> Since all of the different ways that sffo is set by the user, this could do type interpretation to figure it out.

That makes sense, what do you think about doing the type interpretation on the UniValue passed in, instead of relying on the key names? I'd rather do that than try to have this function take a `std::variant` or something like that

Reply:
I think UniValue type interpretation would be fine.

Reply:
Done